package.json

```diff
@@ -1,80 +1,105 @@
 {
-  "postinstall": "install-app-deps",
-  "build": {
-    "asar": false,
-    "directories": {
-      "app": "src/main"
-    },
-    "appId": "io.lbry.LBRY",
-    "mac": {
-      "category": "public.app-category.utilities",
```

Comment:
Where did this line go? Does it not do anything?

Reply:
The `electron-builder` configuration has been moved into its [own file](https://github.com/lbryio/lbry-app/blob/restructuring/electron-builder.json).

I removed the `category` line since the app isn't really a utility compared to the Mac apps categorized as utilities on the App Store. Maybe the category "video" would be more appropriate? https://developer.apple.com/library/content/documentation/General/Reference/InfoPlistKeyReference/Articles/LaunchServicesKeys.html#//apple_ref/doc/uid/TP40009250-SW8

Reply:
Please retain this setting. If you want to change it to video or entertainment I am fine with that.